tools/cassandra/handler.go

```diff
@@ -72,7 +71,7 @@ func verifyCompatibleVersion(
 		return nil
 	}
 
-	if ds.NoSQL.PluginName != cassandra_db.PluginName {
```

Comment:
It's causing another cycle dependency. And since this file will be refactored soon(after the refactor the cycle dep should be gone) so I don't want to spend time on fixing the dependency just for this constant here. I will leave a comment to explain why I don't use the constant.